string[] names
bool use_sensors
---
float32[] joint_angles
